nix/offchain.nix

```diff
@@ -232,6 +232,14 @@ in
                 type = types.path;
               };
 
+              strictComp = lib.mkOption {
```

Comment:
Can we call it `compileStrict`?

Reply:
Done! I named it `strictComp` because it's the argument name used in the CTL code.
However wherever `liqwid-nix` calls external utils like here it would nice having also another option `extraArgs` that let you specify all the other arguments, if you agree I could open a another PR.

Reply:
Yeah, this makes sense! It would be a useful escape hatch to have, and it's in the spirit of liqwid-nix, in general.